import.lua

```diff
@@ -5,22 +5,51 @@
 
     modules { -- Can be `module 'module_name'` as well
         'module_name', -- This is the name of the file in the modules folder in qbx-core without the .lua
-        'client:module_name' -- You can specify the side it should be loaded on like this
+        'client:module_name', -- You can specify the side it should be loaded on like this
+        'resourceName:module_name', -- You can also specify the resource it should be loaded from if you don't want to load it from qbx-core
+        'resourceName:client:module_name' -- The resource loading also accepts a side to load on
     }
 ]]
 
-local resourceName = 'qbx-core'
+local resources = {}
 local isServer = IsDuplicityVersion()
 
+local function getSide(value)
+    return value == 'client' and 'client' or value == 'server' and 'server' or value == 'shared' and 'shared' or 'none'
+end
+
+local function checkSide(value)
+    return value == 'client' and not isServer or value == 'server' and isServer or value == 'shared'
+end
+
+for i = 0, GetNumResources() - 1 do
+    local resource = GetResourceByFindIndex(i)
+    if GetResourceState(resource) == 'started' then
+        resources[resource] = true
+    end
+end
+
 for i = 0, GetNumResourceMetadata(resourceName, 'module') - 1 do
```

Comment:
Is this function only called once to get the number of resource metadata? Or is it called on every iteration?

Reply:
GetNumResourceMetadata will only be called once to get the number to reach until it needs to end